source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,124 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v7.0 - Major Release](#release-v7-0-major-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
-- [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.0 - [Major Release](https://docs.mattermost.com/upgrade/release-definitions.html#major-release)
+
+**Release Day: 2022-06-15**
+
+### Important Upgrade Notes
+ - MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in the ``FileInfo`` table. For MySQL, it takes around 19 seconds for a table of size 0.7M rows. The time required for PostgreSQL is negligible.
```

Comment:
```suggestion
 - MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in the ``FileInfo`` table. For MySQL, it takes around 19 seconds for a table of size 700,000 rows. The time required for PostgreSQL is negligible.
```

Reply:
Does the time increase linearly, or at another rate? E.g. will someone with twice that number of rows have twice the upgrade time or more/less?

Reply:
@agarciamontoro / @agnivade to confirm.

Reply:
Meant to ping @harshilsharma63 here.

Reply:
This is a common question to which there is no good answer. Theoretically speaking, the time should increase linearly, but there are just so many variables to consider that it's impossible to make any predictions. E.g. for our v6.0 migrations, for a certain customer it took a far longer time for a dataset 5x the size.

Historically, we have just mentioned the observed time and let customers make a knowledgeable decision.